packages/components/Avatar/src/useAvatar.ts

```diff
@@ -38,16 +38,16 @@ export const useAvatar = (props: AvatarProps): AvatarInfo => {
 
   const showRing = active === 'active' && activeAppearance === 'ring';
   const showBadge = (!active || active === 'unset') && !!badge && !!badge.status;
-  const accessibilityText = `${name || ''}${showBadge ? `, ${badge.status}` : ''}`;
-
+  const accessibilityText = `${accessibilityLabel ? `${accessibilityLabel}` : undefined || `${name || ''}`}
+   ${showBadge ? `,${badge.status}` : ''}${active != undefined ? active : ''} ${badge?.outOfOffice ? 'Out of Office' : ''}`;
```

Comment:
This would only work for English. We don't support localized stings in FURN, so you'll have to either have the client feed in this information or use a native prop which will localize the information for you.

Reply:
Yeah this is one of the reasons we still have the iOS native native Avatar around :/ 
We either have to set up some sort of localization pipeline for FURN or instruct clients to localize this themselves. I hadn't found a way to concatenate info in an accessibility label that's internationalization friendly

Reply:
Totally, make sense. 
i think the issue is across many of our components, We need to have some structure how we should handle accessibility labels and localisation.

One challenge i see here (may exist with other components as well) is sometime we map the component to types support 'active' , 'inactive' , or any other property we used from types.

Reply:
I think in the short term the guidance we give is that clients feed in the full string. A longer term solution should be worked on, but we don't have bandwidth to support a proper solution for at least the next few months.